middleware/legacy/rules/entity.js

```diff
@@ -1,13 +1,13 @@
 // Redirect legacy entity page URLs
 
 export default (route) => {
-  const pattern = /^\/portal(\/[a-z]{2})?(\/explore\/(people|topics))(\/[0-9]+)/;
+  const pattern = /^\/portal(\/[a-z]{2})?(\/explore\/(people|topics|times))(\/[0-9]+)/;
```

Comment:
This should not add "times" but "periods" which is what the classic portal used.